packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -378,7 +380,13 @@ describe('CheckoutModule', () => {
                 // Use a new copy per test so any mutations do not affect subsequent tests
                 checkoutDeliveryCopy = Object.assign(checkoutDelivery);
 
-                axios.get = jest.fn(() => Promise.resolve({ data: checkoutDeliveryCopy }));
+                axios.get = jest.fn(url => {
+                    if (url === 'http://localhost/customer') {
```

Comment:
Repeated string `http://localhost/customer`.

Reply:
This conditional in the `beforeEach` is an indicator that it should probably have a default value, and then you can override it in the test(s) that you need something different, in my humble opinion.

i.e. (pseudocode):
```
beforeEach()
  axios.get = ... return Promise.resolve({ data: customer });


it('myTest')
  axios.get = ... return Promise.resolve({ data: checkoutDeliveryCopy });
```

Reply:
Refactored